org.gnome.IconPreview.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id" : "org.gnome.IconPreview",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gnome-icon-preview",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
```

Comment:
Missing `cleanup` section for things like `/include`, `/lib/pkgconfig`, etc.

Reply:
Done